backend/models/quoteModel.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose')
+
+const Schema = mongoose.Schema;
+
+const quoteSchema = new Schema({
+    quote: {
+        type: String,
+        required: true,
+    },
+    note:{
+        type: String,
+        required: true,
+    },
+    tag:{
+        type: String,
+    },
+    wantToRmb: {
```

Comment:
strange variable name. what does it mean?

Reply:
should I just use fav?

Reply:
i don't know what it means